e-commerce_fake-store/src/components/product-card/index.js

```diff
@@ -0,0 +1,89 @@
+import "./index.css";
+import { Link } from "react-router-dom";
+import React, { useState } from "react";
+
+const Product = ({
+  id,
+  title,
+  price,
+  description,
+  category,
+  image,
+  rating,
+  count,
+  variant,
+  itemquantity,
+}) => {
+  const [quantity, setquantity] = useState("");
+ 
+  const cartcount = Number(localStorage.getItem("cartcount"));
+  const carttotal = Number(localStorage.getItem("carttotal"));
+  const cartitem = Number(localStorage.getItem("cartitem"));
+  let cartDetails=[];
+  const handleClick = (e) => {
+    if (quantity < 1) {
+      alert("Please select a valid quantity");
+    } else {
+      localStorage.setItem("cartcount", cartcount + Number(quantity));
```

Comment:
As already discussed in previous sessions, why multiple localStorage has been set for the data that will be a single object?

Create a single object for all info that needs to be stored and then maintain that object in the local storage

@shubhamshukla91